app/javascript/components/lease_ups/leaseUpAppFirstComeFirstServedModel.js

```diff
@@ -4,12 +4,12 @@ export const buildLeaseUpAppFirstComeFirstServedModel = (application) => {
   return {
     application_id: application.id,
     application_number: application.name,
-    first_name: applicant.first_name,
-    last_name: applicant.last_name,
-    phone: applicant.phone,
-    email: applicant.email,
-    mailing_address: applicant.mailing_address,
-    residence_address: applicant.residence_address,
+    first_name: applicant?.first_name,
```

Comment:
@alulabeshue-sfgov we actually probably don't need this because I'm filtering out the applications without applicants first